core/renderer.go

```diff
@@ -0,0 +1,54 @@
+package core
+
+import (
+	"io/ioutil"
+	"strings"
+
+	"github.com/AlecAivazis/survey/terminal"
+	"github.com/chzyer/readline"
+)
+
+// basic readline config that does not put terminal in Raw mode
+// so that basic editing works (backspace, arrow keys etc).  This
+// is used by all text-input prompt types
+var SimpleReadlineConfig = &readline.Config{
```

Comment:
Yeah, the config from `terminal.GetReadline()` still puts the tty into raw mode which prevents  the default terminal echo.  The `SimpleReadlineConfig` leaves the tty in raw mode by stubbing out the Enter/ExitRawMode callbacks.  Without the `SimpleReadlineConfig` readline does all the output control which erases my prompt via `033[J` in the runebuffer (this is the stuff I tried to work around in Password) .  The only other way to keep the readline-rawmode and get this to work is to implement an OnChange function that basically re-implements the echo functionality from your tty that is disabled by readline.  I went that way first but found that I would also have to re-implement all the native tty functionality for editing (ie backspace, arrow keys to type over content didnt work).